mixer/pkg/runtime/dispatcher/session_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	tpb "istio.io/api/mixer/adapter/model/v1beta1"
 	mixerpb "istio.io/api/mixer/v1"
+	v1 "istio.io/api/mixer/v1"
```

Comment:
why do we have the same import with two differing aliases ?

Reply:
good question... not needed and removed